src/photos/ducks/clustering/files.js

```diff
@@ -71,24 +50,29 @@ export const getFilesByAutoAlbum = async album => {
   album._type = DOCTYPE_ALBUMS
   let files = []
   let next = true
-  let skip = 0
+  let startDocid = ''
+
   while (next) {
-    const result = await cozyClient.data.fetchReferencedFiles(album, {
-      skip: skip,
-      wholeResponse: true
-    })
+    const key = [DOCTYPE_ALBUMS, album._id]
+    const cursor = [key, startDocid]
+    const result = await cozyClient.data.fetchReferencedFiles(
+      album,
+      { cursor },
+      'id'
```

Comment:
I think this should be `_id`, `id` shouldn't even be in the couchdb.

Reply:
Actually, this is not the id of the document, but the sort type, see https://github.com/cozy/cozy-client-js/blob/3326271ca0c5796b1c2d660731d7f20e5bb1c845/src/relations.js#L33